packages/core/content-releases/shared/validation-schemas.ts

```diff
@@ -3,22 +3,22 @@ import * as yup from 'yup';
 export const RELEASE_SCHEMA = yup
   .object()
   .shape({
-    name: yup.string().trim().required(),
+    name: yup.string().trim().required('Required field'),
```

Comment:
we have generic translations for this sort of thing coming from the helper-plugin:

https://github.com/strapi/strapi/blob/a76a2ee7a9a66bc0233e678ca062086c34855a5c/packages/core/helper-plugin/src/utils/translatedErrors.ts#L10